main.cpp

```diff
@@ -115,19 +139,19 @@ struct <#structname2#>
 
 int main()
 {
-    T <#name1#>( , );                                             //6
-    T <#name2#>( , );                                             //6
+    T t1(9, "text1");                                             //6
+    T t2(3, "text2");                                             //6
     
-    <#structName1#> f;                                            //7
-    auto* smaller = f.compare( , );                              //8
+    T* f;                                            //7
```

Comment:
What name did you use when you replaced `<#structName1#>` on line 64? 
Use that same name here. 
- [ ] revise